product_configurator_sale/models/sale.py

```diff
@@ -64,12 +64,26 @@ def _compute_price_unit(self):
         for line in self:
             if line.config_session_id:
                 account_tax_obj = self.env["account.tax"]
+                config_price = line.config_session_id.price
+                # context = self.env.context
+                # add config price to context
+                # self = self.with_context(config_price=config_price)
+
+                # import ipdb; ipdb.set_trace()
+                if line.order_id.pricelist_id.discount_policy == "with_discount":
```

Comment:
```suggestion
                if line.order_id.pricelist_id.discount_policy == "with_discount" and line.product_id:
```
This will solve issues with failing tests.